app/js/home/home.html

```diff
@@ -1 +1,47 @@
-{{vm.hello}}
+<div align="center">
+
+    <!-- Greeting -->
+    <div>
+        <h1>{{vm.hello}}</h1>
+    </div>
+
+    <!-- Counter -->
+    <fieldset>
+        <legend>COUNTER</legend>
+        <div>
+            <div>
+                <h3>Controller functionality example</h3>
+            </div>
+            <div>
+                <div>{{vm.counter}}</div>
+                <div>
+                    <button ng-click="vm.incrementCounter()">INCREMENT</button>
+                </div>
+            </div>
+        </div>
+    </fieldset>
```

Comment:
I would get rid of this example... It shows a bad pattern of including a business logic in a controller. Really controller should be only a layer on top of a service to handle UI.